src/skas/core.py

```diff
@@ -189,7 +189,11 @@ def get_syllables(word_list):
     syllabified_words = []
     for word in word_list:
         if word.is_alpha:
-            pos, tag = word.tag_.split("__")
+            if '__' in word.tag_:
+                pos, tag = word.tag_.split('__')
+            else:
+                pos = ""
```

Comment:
I think this line should be `pos = word.pos_ or word.tag_ or ""` better.

Reply:
Ok!

Reply:
`        if word.is_alpha:
            if '__' in word.tag_:
                pos, tag = word.tag_.split('__')
            else:
                pos = word.pos_ or ""
                tag = word.tag_ or ""`